docs/proposals/scc-check.md

```diff
@@ -0,0 +1,89 @@
+## Problem
+It is impossible for a user to know whether the `PodSpec` he's describing will actually be allowed by the current SCC rules.
+This becomes even more problematic when trying to reason about whether a `DeploymentConfig` or `ReplicationController` or other resource containing a `PodSpec` 
+that a system component will eventually create will work.
```

Comment:
> In the future when we have dry-run of admission is this obviated?

Yes, but the problem is so annoying and intractable now, I think its worth the API.  I don't think that we'll get a dry-run of admission as quickly as we can get this API.